scala-thoughts.md

```diff
@@ -0,0 +1,55 @@
+## Scala thoughts
+
+#### Likes
+
+Overall, my impression of Scala is that it is very intuitive.
+To me, it seems most similar to Python, but with static typing.
+
+I personally like the static typing because it can make the code a lot more
+readable and make it easier to understand what the input and output of each
+method is.
+
+The code seems a lot more concise than a lot of other languages I've worked
+with.
+For example, the `if` `then` `else` syntax only has to have those keywords
+without any braces, parentheses, or colons.
+Another example is inserting values into strings just requires `s` before and
+`$` in the string making it very readable.
```

Comment:
I liked this feature too! It makes string manipulation intuitive and readable, like you said.